.github/workflows/reuse.yml

```diff
@@ -21,7 +22,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.sha || '' }}
```

Comment:
This looks ok to me. IIRC we found that the merge "branch" is better usually, but it's not available on the `opened` trigger. I'd guess this will be good enough though. It will likely suffer from similar issues we saw of when a PR branch is behind the target branch, and errors were in the target at the time of branching but since fixed.

I'd say it's worth merging it like this because it's a much easier way of doing it, and if it causes problems, then try to something with the merge branch.

Reply:
For this test (run `reuse lint`) I think the PR's branch suffices, instead of the merge commit. Assuming the `reuse` tool doesn't suddenly has breaking changes :)